assets/files.wxs

```diff
@@ -1930,6 +1930,11 @@
             <File Id="filFF877991123D49A58FD15CB83AD44BA7" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Delay works
```powershell
Expected the actual value to be less than 3, but got 3.3178178999999997.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 150
150:             $result1.TotalSeconds | Should -BeLessThan 3
```